.github/workflows/ci.yml

```diff
@@ -44,14 +44,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # TODO: add windows which does not support container
-        os: [ ubuntu-22.04 ]
+        os:
+          - ubuntu-22.04
+          - windows-latest
```

Comment:
should the windows os version be fixed a certain version?

Reply:
Pinned the Windows version to 2022, let's test it out.